assets/css/main.css

```diff
@@ -13,10 +13,8 @@ body {
   color: #404040;
   position: relative;
   background-color: {{ site.page-col }};
-  {% if site.page-img %}
-  background-image: url({{ site.page-img }});
-  background-attachment: fixed;
-  {% endif %}
+  {% if site.page-img %}background-image: url({{ site.page-img }});
```

Comment:
Just to check:  is this change deliberate or accidental?  It looks like some lines were joined but no actual source changed.

Reply:
The if statements don't actually render, so all I did here was make the actual thing render more like it would without the if statements (without a newline). For example if we had:

``
line 1
{% if content %}
line 2
{% endif %}
``
That would render on the page as

```
line 1

line 2
```

If you do it like this:

``
line 1
{% if content %}line 2{% endif %}
``

Then you would get

```
line 1
line 2
```

It you look at a lot of jekyll sites, they have a ton of newlines just because people don't think about this. So, it's just an effort to clean up what is rendered of the newlines, the change doesn't change anything.

Reply:
You can actually see it's pretty severe in our site :P view-source:https://us-rse.org/

Reply:
![image](https://user-images.githubusercontent.com/814322/59629555-1e47bc00-9111-11e9-9cda-7e14d28ea574.png)